integrations/airflow/Dockerfile.tests

```diff
@@ -2,10 +2,12 @@ FROM python:3.6-slim
 RUN mkdir /app
 ADD airflow/setup.py /app
 ADD airflow/README.md /app
-COPY common/marquez /app/marquez
+COPY common /tmp/marquez-integration-common
+RUN cd /tmp/marquez-integration-common && ls . && pip wheel --wheel-dir=/tmp/marquez-integration-common/wheel .
```

Comment:
_Minor_: Do you still need run the `ls .` cmd?

Reply:
nope